tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
-  "include": ["src"],
+  "include": ["src/**/*"],
+  "exclude": ["src/**/*.test.ts"],
   "compilerOptions": {
-    "target": "es2018",
+    "target": "es2020",
```

Comment:
I think we should bump the Node to 14 in `package.json`'s "engines" property since ES2020 is our target right now.

Reply:
Done